kiwi_stackbuild_plugin/tasks/system_stash.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2021 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms owf the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+"""
+usage: kiwi-ng system stash -h | --help
+       kiwi-ng system stash --root=<directory> --target-dir=<directory>
+           [--tag=<name>]
+           [--maintainer=<name>]
+           [--name=<container_name>]
+       kiwi-ng system stash help
+
+commands:
+    stash
+        create a container from the given root directory
+    stash help
+        show manual page for stash command
+
+options:
+    --root=<directory>
+        the path to the root directory, usually the result of
+        a former system prepare or build call
+    --target-dir=<directory>
+        the target directory to store the container files
+    --tag=<name>
+        the tag name for the container.
+        By default set to: latest
+    --maintainer=<name>
+        the maintainer name used in the history/author and label
+        section of the container metadata.
+        By default set to: KIWI
+    --name=<container_name>
+        the name of the container as it appears in the registry.
+        By default set to: kiwi-stash-DATE
```

Comment:
Similar here, why not using something like `<former-image-name>-DATE`? I know the name would require some sanitization to be a valid image name (e.g. no capital characters) and that this is super annoying, but I think this would be a nice default specially for the rebuild concept.

Reply:
With the name I put on a convention and hope this makes sense. I leave it up on the review if this should change. Actually the invocation of the command is much easier than before

Reply:
When implementing the actual build using a stash the same convention criteria can apply